src/main/java/org/ssu/edu/teachua/ui/components/modal/AddLocationComponent.java

```diff
@@ -55,57 +56,66 @@ public AddLocationComponent(WebDriver driver) {
         super(driver);
     }
 
+    @Step("Close 'Додати локацію' window")
     public AddCenterMainInfoComponent pressCloseAddLocationWindow() {
         waitForElementToBeClickable(closeAddLocationWindowButton).click();
         return new AddCenterMainInfoComponent(driver);
     }
 
+    @Step("Enter location name")
```

Comment:
add parameters to step ` @Step("Enter location name {locationName}")`

Reply:
Done